sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/it/ChangeStreamTestPipelineOptions.java

```diff
@@ -31,7 +31,7 @@
   void setProjectId(String value);
 
   @Description("Instance ID to write to in Spanner")
-  @Default.String("beam-test")
+  @Default.String("changestream")
```

Comment:
We can not change this, since the tests run in beam's instance

Reply:
Ah, I did not mean to change this.